alws/models.py

```diff
@@ -194,6 +194,7 @@ class Build(Base):
         secondaryjoin=(BuildDependency.c.build_dependency == id)
     )
     mock_options = sqlalchemy.Column(JSONB)
+    pgp_key_id = sqlalchemy.Column(sqlalchemy.VARCHAR(length=50), nullable=False)
```

Comment:
Here you say about VARCHAR, but in alembic migration, the field is a Text. Is it OK? There different types for DB.